public/src/components/Manager/Manager.vue

```diff
@@ -7,8 +7,8 @@
     </v-fade-transition>
     <v-toolbar flat class="elevation-1" app color="secondary" clipped-left>
       <v-toolbar-items class="ml-0">
-        <v-btn flat color="primary" @click="$router.push('/')" class="main-mph">
-         <img src="../../assets/mph_logo.png">
+        <v-btn flat color="primary" @click="$router.push('/')" class="main-mph-btn">
```

Comment:
we'll probably want to setup some simpler asset routing in the future.. all good tho 👍